src/Sulu/Bundle/ContactBundle/Behat/ContactContext.php

```diff
@@ -22,12 +22,16 @@
  */
 class ContactContext extends BaseContext implements SnippetAcceptingContext
 {
+    private static $emailTypeId = 100;
```

Comment:
Because the `AssignedGenerator` is used on the email-types. That's because in the source code we have hardcoded ids for different types (also for the system-collection type in the media-bundle).